http/server.ts

```diff
@@ -122,6 +122,26 @@ export class ServerRequest {
   conn: Conn;
   r: BufReader;
   w: BufWriter;
+  private _cookie: object;
+
+  get cookie(): object {
+    if (this._cookie) {
+      return this._cookie;
+    }
+    if (this.headers.has("Cookie")) {
+      const out: {[key:string]: string} = {};
```

Comment:
I found that there are several places that reference the `object` type. A type should be declared independently.

eg.

```typescript
type KeyValue = { [key: string]: string };

private _cookie: KeyValue

get cookie(): KeyValue
```

Reply:
The type may have to be exported too so.

Reply:
yes. that's good. then exposed it.